plugins/infinispan90/src/main/java/org/radargun/service/Infinispan90HotRodOperations.java

```diff
@@ -0,0 +1,40 @@
+package org.radargun.service;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.StreamingRemoteCache;
+import org.radargun.traits.StreamingOperations;
+
+public class Infinispan90HotRodOperations extends Infinispan72HotRodOperations implements StreamingOperations {
+
+   public Infinispan90HotRodOperations(InfinispanHotrodService service) {
+      super(service);
+   }
+
+   public <K, V> StreamingCache<K> getStreamingCache(String cacheName) {
+      HotRodCache<K, V> cache = super.getCache(cacheName);
+      return new StreamingHotRodCache<K, V>(cache.noReturn, cache.forceReturn, cache.noReturn.streaming());
+   }
+
+   protected class StreamingHotRodCache<K, V> extends HotRodOperations.HotRodCache<K, V>
```

Comment:
Seeing that StreamingRemoteCache is just extension (not in programming sense) of the normal RemoteCache and all configuration applies to it, it did seem reasonable at the time, but you are right.